src/Symfony/Component/Asset/CompilerPass/GitCommitHashCompilerPass.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Asset\CompilerPass;
```

Comment:
I just think that because components can be used independently, there is Asset component looks better for this. Also as if this PR will be accepted, there will be more strategies, and all this stuff is not related to DI component.